service/src/main/java/uk/gov/hmcts/reform/fpl/model/notify/allocatedjudge/AllocatedJudgeTemplateForC2.java

```diff
@@ -0,0 +1,12 @@
+package uk.gov.hmcts.reform.fpl.model.notify.allocatedjudge;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public final class AllocatedJudgeTemplateForC2 extends SharedAllocatedJudgeTemplate {
+    private String callout;
+    private String respondentLastName;
+
```

Comment:
minor: unnecessary white space in here and your other Template models. Won't note them all.

Reply:
removed